products/models/product.py

```diff
@@ -18,5 +18,8 @@ class Product(models.Model):
 
     orders = models.ManyToManyField('Order', through='OrderProduct')
 
+    create_at = models.DateTimeField(auto_now_add=True)
+    update_at = models.DateTimeField(auto_now=True)
+
```

Comment:
pls add relatiton to the store through model StoreInventory as we did in the lesson between order and product.